autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -239,6 +239,22 @@ export function updateCourseCard(index: number, courseCard: CourseCardOptions, t
   };
 }
 
+export function toggleSelected(courseCardId: number, secIdx: number):
```

Comment:
I'm fine with you moving this here since we don't need a selector in the `SectionInfo` component, and it abstracts out the logic a bit from the component itself